packages/graphql/src/schema/generation/aggregate-types.ts

```diff
@@ -74,6 +74,36 @@ export function withAggregateSelectionType({
     return aggregateSelection;
 }
 
+/** Top level count */
+export function getCountType(composer: SchemaComposer): ObjectTypeComposer {
+    const CountFieldName = "Count";
+    return composer.getOrCreateOTC(CountFieldName, (countField) => {
+        countField.addFields({
+            nodes: {
+                type: new GraphQLNonNull(GraphQLInt),
+                resolve: (source, args, context, info) => numericalResolver(source, args, context, info),
```

Comment:
I believe that the extra arrow function is not needed here.
```suggestion
                resolve: numericalResolver(source, args, context, info),
```

Reply:
To be honest, I'm not 100% sure if we need this arrow function, but this pattern `resolve: (source, args, context, info) => numericalResolver(source, args, context, info),` is what we use elsewhere in the codebase for numericalResolvers, if we prefer removing the arrow functions I'd rather do it and test it in a separate PR in all the resolvers

Reply:
I think is safe to remove it, is already used without the arrow function:
`packages/graphql/src/schema/aggregations/field-aggregation-composer.ts:99`
`packages/graphql/src/schema/to-compose.ts:88`
!